backend/src/main/java/com/google/blackswan/mock/SimpleAnomalyGenerator.java

```diff
@@ -34,22 +33,48 @@
 import java.nio.charset.StandardCharsets;
 import java.io.ByteArrayInputStream;
 import com.google.common.collect.ImmutableList;
+import com.google.blackswan.mock.filesystem.*;
 
 /** Generate list of anomalies based on data in the csv file using average and threshold. */
 public class SimpleAnomalyGenerator implements AnomalyGenerator {
   private static final String DATA_FILE_LOCATION = "/sample-ramen-data.csv";
-  private static final String EXCEPTION_MESSAGE = "Invalid row format.";
-  private static final String COMMA_DELIMITER = ",";
   private static final int THRESHOLD = 13;
   private static final int NUM_POINTS = 5;
+  private static final String DEFAULT_METRIC = "interest";
+  private static final String DEFAULT_DIMENSION = "ramen";
 
   private final ImmutableList<Anomaly> anomalies;
 
+  /** TODO: Let generators take in parameters of dimension/metric names. */
+  public static SimpleAnomalyGenerator createGenerator() {
+    return new SimpleAnomalyGenerator(
+      // For [push] to git, always use LocalFileSystem, as CloudFileSystem will fail 
+      // unit test without access to key.json. 
+      LocalFileSystem.createSystem().getDataAsStream(DEFAULT_METRIC, DEFAULT_DIMENSION),
+      THRESHOLD,
+      NUM_POINTS
```

Comment:
could these also be specified (not default) in the future?

Reply:
yes, definitely. do you think it'll be good to have the user specify (in the future if we have time) ?

Reply:
Maybe not user specified, but passed in as parameters
Just a thought lol. we probably won't get to that point